src/templates/article.tsx

```diff
@@ -84,6 +164,16 @@ export default function Article(props: PropsArticle): ReactElement {
           </p>
         </article>
       </div>
+      <Text as="h2" className="mb-12 text-center">
+        Lectures Liées
+      </Text>
+      <Text
+        as="h3"
+        className="flex flex-col m-auto mb-12 text-center w-2/3 space-y-8 "
+      >
+        {previous !== null && <Previous {...props} />}
```

Comment:
I think the api here could be
```javascript
<ReadMoreLink uid={nextUid} imgUrl={nextImgUrl} title={nextTitle} >
<ReadMoreLink uid={previousUid} imgUrl={previousImgUrl} title={previousTitle} >
```

that way we have only one component to maintain